src/odemis/driver/test/xt_client_test.py

```diff
@@ -199,6 +199,15 @@ def test_move_stage_rot_tilt(self):
                                (abs_pos["rz"] + rel_pos["rz"]) % (2 * math.pi), places=4)
         self.assertAlmostEqual(self.stage.position.value["rx"],
                                (abs_pos["rx"] + rel_pos["rx"]), places=4)
+
+        # Relative move in R < 0
+        f = self.stage.moveAbs({"rz": 0})
+        rel_pos = {"rz": -0.5} # rad
+        f = self.stage.moveRel(rel_pos)
+        f.result()
+        self.assertAlmostEqual(self.stage.position.value["rz"] % (2 * math.pi),
+                               (0 + rel_pos["rz"]) % (2 * math.pi), places=4)
```

Comment:
can you add a comment on why you add the zero in: `(0 + rel_pos["rz"])`

Reply:
I wanted to keep the same formula as the test above, which is old_pos + rel_pos. However, that's probably more confusing than helpful... will remove it.